tiledb/sm/serialization/tiledb-rest.capnp

```diff
@@ -4,1370 +4,1366 @@ using Json = import "/capnp/compat/json.capnp";
+  denseReader @16 : QueryReader;
+#denseReader contains data needed for continuation of incomplete dense reads \
+    with dense reader
 
-    delete @17 :Delete;
-    # delete contains QueryCondition representing deletion expression
+  delete @17 : Delete;
+#delete contains QueryCondition representing deletion expression
 
-    writtenFragmentInfo @18 :List(WrittenFragmentInfo);
-    # Needed in global order writes when WrittenFragmentInfo gets updated
-    # during finalize, but doesn't end up back on the client Query object
+  writtenFragmentInfo @18 : List(WrittenFragmentInfo);
+#Needed in global order writes when WrittenFragmentInfo gets updated
+#during finalize, but doesn't end up back on the client Query object
 
-    writtenBuffers @19 : List(Text);
-    # written buffers for partial attribute writes
+  writtenBuffers @19 : List(Text);
+#written buffers for partial attribute writes
 
-    orderedDimLabelReader @20 :QueryReader;
-    # orderedDimLabelReader contains data needed for dense dimension label reads.
+  orderedDimLabelReader @20 : QueryReader;
+#orderedDimLabelReader contains data needed for dense dimension label reads.
 
-    channels @21 :List(QueryChannel);
-    # channels contains the list of channels (streams of data) within a read
-    # query. It always contains at least one element, the default channel.
+  channels @21 : List(QueryChannel);
+#channels contains the list of channels(streams of data) within a read
+#query.It always contains at least one element, the default channel.
 }
 
 struct NonEmptyDomain {
-  # object representing a non-empty domain
+#object representing a non - empty domain
 
-  nonEmptyDomain @0 :DomainArray;
-  # Non-empty domain of array
+  nonEmptyDomain @0 : DomainArray;
+#Non - empty domain of array
 
-  isEmpty @1 :Bool;
-  # Is non-empty domain really empty?
+  isEmpty @1 : Bool;
+#Is non - empty domain really empty ?
 
-  sizes @2 :List(UInt64);
-  # Number of elements in DomainArray for var length
+  sizes @2 : List(UInt64);
+#Number of elements in DomainArray for var length
 }
 
 struct NonEmptyDomainList {
-  # object representing non-empty domain for heterogeneous or string dimensions
-  nonEmptyDomains @0 :List(NonEmptyDomain);
+#object representing non - empty domain for heterogeneous or string dimensions
+  nonEmptyDomains @0 : List(NonEmptyDomain);
 }
 
 struct AttributeBufferSize {
-  # object representing a buffer size of an attribute
+#object representing a buffer size of an attribute
 
-  attribute @0: Text;
-  # name of attribute
+  attribute @0 : Text;
+#name of attribute
 
-  offsetBytes @1: UInt64;
-  # size (in bytes) of offset buffer
+  offsetBytes @1 : UInt64;
+#size(in bytes) of offset buffer
 
-  dataBytes @2: UInt64;
-  # size (in bytes) of data buffer
+  dataBytes @2 : UInt64;
+#size(in bytes) of data buffer
 
-  validityBytes @3: UInt64;
-  # size (in bytes) of data buffer
-}
-
-struct MaxBufferSizes {
```

Comment:
This structure is not used by the rest of the capnp file. Removing it looks safe.